app/src/main/java/com/arygm/quickfix/model/account/AccountRepositoryFirestore.kt

```diff
@@ -136,4 +147,38 @@ open class AccountRepositoryFirestore(private val db: FirebaseFirestore) : Accou
           onFailure(exception)
         }
   }
+
+  override fun uploadAccountImages(
+      accountId: String,
+      images: List<Bitmap>,
+      onSuccess: (List<String>) -> Unit,
+      onFailure: (Exception) -> Unit
+  ) {
+    val accountFolderRef = storageRef.child("accounts").child(accountId)
+    val uploadedImageUrls = mutableListOf<String>()
+    var uploadCount = 0
+
+    images.forEach { bitmap ->
+      val fileRef = accountFolderRef.child("image_${System.currentTimeMillis()}.jpg")
+
+      val baos = ByteArrayOutputStream()
+      bitmap.compress(Bitmap.CompressFormat.JPEG, 90, baos) // Compression qualité 90
```

Comment:
Keep the comments in English in the future please 😉

Reply:
Thank you for your feedback I will do it next time